_data/releases.csv

```diff
@@ -309,3 +309,4 @@ v22.2.0-rc.1,v22.2,2022-11-07,false,False,Testing,False,go1.18,3a8bf9fd51f08df55
 v22.2.0-rc.2,v22.2,2022-11-14,false,False,Testing,False,go1.18,b1df37d0c1aabc62dd2b93edd727db46ee00d741,cockroachdb/cockroach-unstable,true,true,true,true,true
 v22.1.11,v22.1,2022-11-14,false,False,Production,False,go1.18,22fa382362ff4ec680d82eef28dcbbaa932942ae,cockroachdb/cockroach,true,false,false,false,true
 v22.2.0-rc.3,v22.2,2022-11-21,false,False,Testing,False,go1.18,77667a1b0101cd323090011f50cf910aaa933654,cockroachdb/cockroach-unstable,true,true,true,true,true
+v22.2.0,v22.2,2022-12-05,false,False,Production,False,go1.18,77667a1b0101cd323090011f50cf910aaa933654,cockroachdb/cockroach,true,true,true,true,true
```

Comment:
@nickvigilante - I updated releases.csv and versions.csv manually for GA release notes. Can you take a look just to make sure I edited them correctly? Thanks!

Reply:
Follow up: I noticed that the sidebar didn't update to include v22.2 (and remove v21.2). Do I need to update this manually?